useful-scripts/get_data_block_responses/get_data_block_responses.py

```diff
@@ -22,18 +23,38 @@
 Place it in the same directory as this script.
 """
 
-DATA_API_URL = 'https://data.explore-education-statistics.service.gov.uk/api'
+parser = argparse.ArgumentParser(prog="python get_data_block_responses.py",
+                                 description="Used to get and time data block responses from "
+                                             "an environment")
+parser.add_argument("-u", "--url",
```

Comment:
For convenience, would refactor this as `-e`/`--env` (similar to what we're doing with the Robot tests) and lookup the URL from a dictionary of environments to URLs.